Gruntfile.js

```diff
@@ -0,0 +1,71 @@
+module.exports = function (grunt) {
+
+  grunt.loadNpmTasks('grunt-babel');
+  grunt.loadNpmTasks('grunt-contrib-clean');
+  grunt.loadNpmTasks('grunt-contrib-uglify');
+  grunt.loadNpmTasks('grunt-banner');
+
+  var LICENSE = '/****\n * ' +
+    grunt.file.read('./LICENSE.md', {encoding: 'utf-8'}).replace(/\n/ig, '\n * ') +
+    '\n ****/';
+
+  grunt.initConfig({
+    babel: {
+      options: {
+        ast: false,
+        sourceMaps: false,
+        nonStandard: false,
+        compact: 'false',
+        modules: 'umd',
+        moduleId: 'Timsort',
```

Comment:
I prefer sticking to the lower case version. I don't have any other comment, I run benchmarks on your PR and they seem to run faster in any of the array types. I also think we should keep your version of `alphabeticalCompare`.